fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml

```diff
@@ -14,6 +14,19 @@ observabilityIncluded = true
 org="ballerinax"
 name="health.fhir.r4"
 version="${r4.version}"
+repository="local"
+
+[[dependency]]
+org="ballerinax"
+name="health.fhir.r4.parser"
+version="${parser.version}"
+repository="local"
+
+[[dependency]]
+org="ballerinax"
+name="health.fhir.r4.international401"
```

Comment:
Do we use this dependency anywhere in the fhir service? Seems no?

Reply:
Commented out in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/a9c9f5f3a007a3943103b4f0da56d1d682506ad5. This is a transitive dependency coming from the Parser, and this dependency statement is only required when using the local Ballerina central for dependency resolving. It will be removed once the packages are released.